prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -8,11 +8,22 @@ presubmits: # runs on PRs
       context: prow/kyma/kyma-integration
       skip_report: true # from documentation: SkipReport skips commenting and setting status on GitHub.
       max_concurrency: 10
-      labels:
+      labels: &kyma-integration-labels
         preset-sa-vm-kyma-integration: "true"
-      spec:
+        preset-gc-compute-envs: "true"
+        preset-gc-project-env: "true"
+      decorate: true
+      path_alias: github.com/kyma-project/kyma
+      extra_refs: &kyma-integration-extra-refs
+        - org: kyma-project
+          repo: test-infra
+          base_ref: master
+          path_alias: github.com/kyma-project/test-infra
+      spec: &kyma-integration-spec
         containers:
-          - image: eu.gcr.io/kyma-project/snapshot/test/integration:0.0.2 # created by running `docker build -t <image> .` in the integration-job directory.
+          - image: eu.gcr.io/kyma-project/prow/bootstrap:0.0.1
+            command: 
+              - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/provision-vm-and-start-kyma.sh"
```

Comment:
There's a problem here. You are referring to the file that should exist in test-infra _master_ branch, since this is what you're checking out with decorate. Now this file is added in this P/R, so it's not in master.
It means this PR does not work until it's already merged.
How do we test it then?
Please propose a solution for this "chicken or the egg" problem.

Reply:
To test it, you can change `extra_refs` as follows corresponding to the branch this PR was created from: 
```
extra_refs: 
    - org: suleymanakbas91
      repo: test-infra-1
      base_ref: refactoring
      path_alias: github.com/kyma-project/test-infra
```